### ISSUE TITLE
Breaks

### ISSUE THREAD

**Issue by jodischneider** [LibraryCarpentry/lc-shell#45]
The shell lesson needs 1 or more breaks. I'd suggest after episodes 3, 4, and 5.

**Comment by dakane1** [comment]
Hi Jodi,  Breaks are left up to the instructor to determine.  We follow the lesson guidelines from the Carpentries and they don't include breaks within the lessons or episodes.  Do you feel that episodes 3, 4, and 5 are too long?

**Comment by dakane1** [comment]
@jodischneider Hi Jodi, I'm going to go ahead and close this issue.  See earlier response.

**Comment by jodischneider** [comment]
Yes, I don't think lessons should be more than 2 hours. So my suggestion is to revisit these and to try to split them out. 

Even conceptually, this covers more than it needs.
#5 is 90 minutes -- maybe worth a lesson of its own. I think it's separable from just using the shell.
#6 is 60 minutes -- also quite long, especially at the end of a 4.5 hour + breaks workshop.

One possibility:
If we want a text application here (which is nice) it could be scaled back -- for instance, include a 30 minute version of #6, making it 2 hours max.
Then #5 and an advanced version of #6 could be in a separate lesson, again at about 2 hours long.

**Comment by dakane1** [comment]
Hi Jodi,

Thanks for your feedback.  The only way I have taught this is as a full day with two breaks and a lunch break.  Let me talk to the rest of the group and see what they think or what they have heard from others.  We will be meeting again sometime in June.  I personally think that 2 hours would be too short and we wouldn’t be able to cover anything substantial but I want other’s opinions before the group makes a decision.  I will add this to our next agenda.

Best, Danielle

**Comment by jodischneider** [comment]
Daniele,

Consider how you would teach this if you only had half a day. The afternoon content could be seen as a "part II" of the same lesson.

(I'm coming at this as an iSchool instructor, introducing the VERY basics of the command line in a required MSLIS class.) I'd like to have an LC lesson to point students to, but I'd want it to be a bit more granular.

**Comment by dakane1** [comment]
Hi @jodischneider Jodi, 

I will bring up with the other maintainers the possibility of doing a survey of the instructors who teach LC-Shell to get feedback on the length of the workshop.  I think you also need to realize that we are maintaining this lesson for the use of many people who might be using it in different ways and we can’t always adjust the lesson for one individual or for one situation. If the maintainer team does a survey and depending on the responses we will try to make the changes that are for the greater good. 

Best, Danielle

P.S.  I teach the SC: Unix workshop in four hours so I do know what it is like to teach the content in a shortened period.

**Comment by dakane1** [comment]
Hi @jodischneider, the team discussed this topic at our meeting this morning and we decided to help instructors by acknowledging the time requirements for the lesson, how instructors in the past have placed breaks, chosen lessons etc.  The info will be in the instructor notes which we have an issue open to review and update.  Thank you for bringing it up and we hope by adding this kind of info into the instructor notes it will help those who are new to the lesson feel more comfortable in teaching the shell.  We decided at this time to not survey all instructors who teach this lesson for feedback but we will continue to monitor and discuss further.
